_posts/2017-10-23-pair-programming-with-perry.md

```diff
@@ -0,0 +1,91 @@
+</div>
+<br>
+<span class='os-green'>*Interesting. So what do you recommend pair programming on, like what kind of work?*</span>
+<div style='margin-left: 2em;' >
+    I encourage you to try pairing on a full story or card or bug fix or whatever you call it at, where do you work again?
+</div>
+<br>
+<span class='os-green'>*OnShift.*</span>
+<div style='margin-left: 2em;' >
+    Right! As I was saying, it’s important that you pair on something beginning to end if possible. I think that’s where you get the most benefit out of it.
+</div>
+<br>
+<span class='os-green'>*What kind of benefits we talking about?*</span>
+<div style='margin-left: 2em;' >
+    Code quality for starters! You look like you’ve written some terrible code on your own before, so I’m su-
+</div>
+<br>
+<span class='os-green'>*Watch it, Perry, though you aren’t wrong*</span>
+<div style='margin-left: 2em;' >
+    I’m sure you’ve gone through the process of writing a bunch of code, opening a pull request, and then having to either explain your code to your team or having to make a lot of refactors.
+</div>
+<br>
+<span class='os-green'>*Go on..*</span>
+<div style='margin-left: 2em;' >
+    Pairing can help with these issues. When you pair, you always have someone next to you to bounce ideas off of, share in writing the code, make collaborative design decisions, continuous code review..the list goes on. You can’t just wild west it because your pair holds you acountable and vice versa.
+</div>
+<br>
+<span class='os-green'>*Sometimes I wear a cowboy hat when I code*</span>
+<div style='margin-left: 2em;' >
+    That’s kind of dope actually.
+</div>
+<br>
+<span class='os-green'>*Thank you. What makes for an effective pair?*</span>
+<div style='margin-left: 2em;' >
+    First thing is for each pairer..is that a word, I’m goin’ with it, to check their ego at the door. Pairing is not the time to get all high and mighty, nor is it the time to feel like your skills aren’t good enough. Pairing is a collaborative process and you’re going to find areas that you and your pair are both weak in, but you need to roll with it and learn as much as you can as you go. Be open minded and don’t take it personally that they’re a VIM master while you can’t figure out how to close the editor. You also don’t want to hold anything back. What I mean is that if you have an idea, just start typing, explaining yourself as you type. Maybe it’s not right, but it may spark something in your pair’s head and lead to a solution.
```

Comment:
Change 'First thing is for each pairer..is that a word, I'm goin' with it', to check' to 'First thing is for each pairer... is that a word? I'm goin' with it... to check'